style/sheets/actors/_npc.scss

```diff
@@ -5,6 +5,25 @@
         overflow-y: unset;
     }
 
+    .section.tab[data-tab="notes"] {
+        .header-row {
+            display:  flex;
+            .header-group {
+                margin: 0.25rem 0.5rem;
+                display: flex;
+                flex-direction: column;
+                flex: 1;
+                
+                input {
+                    background: none;
+                    border: none;
+                    line-height: 1.5rem;
+                    height: 1.5rem;
+                    padding: 0 0.1rem;
+                }
+            }
+        }
+    }
```

Comment:
If you're redefining these styles, why are they still called `.header-x`?

Reply:
I'm open for suggestions about better naming/structuring of the class names. I didn't want to mess up too much with the scss files, but i guess that it would be possible to "share" in smart way styles for those fields that are inside of character main tab and those ones. Any preferences?

Reply:
`detail-x` is probably fine

Reply:
done

Reply:
details-row and details-cell would ok?

Reply:
Sure, using the same as the header is also fine, so detail-group/row. Doesn't really matter they'll likely get changed when merged with the v13 changes anyway